pages/_template.jsx

```diff
@@ -2,43 +2,77 @@ import React, { Component, PropTypes } from 'react';
 import Headroom from 'react-headroom';
 import { Link } from 'react-router';
 import { prefixLink } from 'gatsby-helpers';
+import classNames from 'classnames';
 
 require('css/_index.scss');
+
+const navbar_height = 54;
+
 export default class Template extends Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+
+    this._handleHamburger = this._handleHamburger.bind(this);
+  }
+
+  _handleHamburger(e) {
+    e.preventDefault();
+    this.setState({
+      open: !this.state.open
+    });
+  }
+
   render () {
     return (
       <div>
-        <Headroom>
-          <div className="navbar">
-            <div className="navbar__link--wrapper">
+        <Headroom wrapperStyle={{
+            maxHeight: navbar_height
+          }}>
+          <div className={classNames("navbar", {
+              "navbar--open": this.state.open
+            })}>
+            <div
+                className={classNames("navbar__hamburger", {
```

Comment:
Is this indentation right? It should be 2 spaces indented from the line above, looks strange